### PR TITLE
398 uplift icons

### LINKED ISSUE #398: Icons: Make new uplift theme icons available
**Is your feature request related to a problem? Please describe.**
Have a way in angular to access the new beta uplift icons.

**Describe the solution you'd like**
- change script to also copy of the uplifted icons
- a new component to import into angular
- a change in the `app.component.html` to `ngIf` based on theme
- update changelog

**Describe alternatives you've considered**
n/a

**Additional context**
Remember to cut a branch from and PR into `5.2.x`

### PR DESCRIPTION
**Pull request by pwpatton**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Added uplift theme, icons, and font to show off beta theme in apps.

**Related github/jira issue (required)**:
Closes #398

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### 5.2.0 Features
 
+- `[DemoApp]` - Added ability to switch to beta of new uplift theme. NOTE: not yet to be included in your released application `PWP`  ([#398](https://github.com/infor-design/enterprise-ng/issues/398))`
+- `[IconsUplift]` - Added SohoIconsUpliftComponent to pull in uplift theme icons `PWP`  ([#398](https://github.com/infor-design/enterprise-ng/issues/398))`
```
Comment (fitzorama) [inline docs/CHANGELOG.md]: probably need to remove the trailing `

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +15,8 @@
 
 ### 5.2.0 Chore & Maintenance
 
+- `[IconsExt]` - Removed SohoIconsExtendedComponent as all icons are included in SohoIconsComponent `PWP`  ([#398](https://github.com/infor-design/enterprise-ng/issues/398))`
+
```
Comment (fitzorama) [inline docs/CHANGELOG.md]: same here

File: projects/ids-enterprise-ng/package.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@types/jquery": "3.3.24",
     "@types/d3": "4.13.0",
-    "ids-enterprise": "4.17.0-beta.0",
+    "ids-enterprise": "4.18.0-dev.20190319",
```
Comment (tmcconechy) [inline projects/ids-enterprise-ng/package.json]: I thought we wanted to stay on 4.17 for 5.2.0 ?

Comment (pwpatton) [reply projects/ids-enterprise-ng/package.json]: nice catch, puting that back

**Pull request merged**